smt/applications/ego.py

```diff
@@ -99,6 +99,8 @@ def _initialize(self):
         declare("ydoe", None, types=np.ndarray, desc="Initial doe outputs")
         declare("xlimits", None, types=np.ndarray, desc="Bounds of function fun inputs")
         declare("verbose", False, types=bool, desc="Print computation information")
+        declare("tunnel", 0, types=int, desc="1 to enable tunneling in ei")
```

Comment:
tunnel should be better implemented by a boolean (see verbose above) and maybe renamed enable_tunneling with further explanations in the description (enable the penalization of points that have been already evaluated in EI criterion).